feature_engine/transformation/power.py

```diff
@@ -124,3 +124,34 @@ def transform(self, X: pd.DataFrame) -> pd.DataFrame:
         X.loc[:, self.variables_] = np.power(X.loc[:, self.variables_], self.exp)
 
         return X
+    
```

Comment:
can we add a description of inverse_transform in the init docstrings?

Reply:
It is already there, would you please clarify on this?

def inverse_transform(self, X: pd.DataFrame) -> pd.DataFrame:
        """
        Apply the power inverse transformation to the variables.

        Parameters ....